pkg/objstore/s3/s3.go

```diff
@@ -200,7 +201,9 @@ func (b *Bucket) getRange(ctx context.Context, name string, off, length int64) (
 	// NotFoundObject error is revealed only after first Read. This does the initial GetRequest. Prefetch this here
 	// for convenience.
 	if _, err := r.Read(nil); err != nil {
-		r.Close()
+		//runutil.LogOnErr(nil, r, "s3 get range obj close")
```

Comment:
uncomment and remove the line below.

Reply:
yea, it was just for CI test - fixed now (:

Reply:
And checking the reviewer looked at the code 😛